extension/persistence/relational-jdbc/src/main/java/org/apache/polaris/extension/persistence/relational/jdbc/RelationalJdbcConfiguration.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.extension.persistence.relational.jdbc;
+
+import io.smallrye.config.ConfigMapping;
+import java.util.Map;
+
+@ConfigMapping(prefix = "polaris.relation.jdbc")
+public interface RelationalJdbcConfiguration {
```

Comment:
It is preferable to keep config interfaces used by more generic Polaris modules free from Quarkus dependencies, and them have a sub-interface (mostly empty) inside Quarkus modules with config annotations for build-time wiring.

Reply:
I'm not sure about `RelationalJdbcConfiguration` in `polaris-core` this type is specific to the JDBC persistence, which is an extension. I suppose `quarkus/server` can get this type from the JDBC module. IIRC it depends on JDBC already.

Reply:
Agreed, I liked Alex suggestion, moved this to quarkus/commons https://github.com/apache/polaris/pull/1482#discussion_r2069388646